integration-tests/chopsticks/package.json

```diff
@@ -8,8 +8,8 @@
   "author": "[\"KILT <info@kilt.io>\"]",
   "license": "MIT",
   "devDependencies": {
-    "@acala-network/chopsticks": "0.10.0",
-    "@acala-network/chopsticks-testing": "0.10.1",
+    "@acala-network/chopsticks": "0.12.2",
+    "@acala-network/chopsticks-testing": "0.12.2",
```

Comment:
Mhh since I am using them locally, right now, I thought we might want to use the latest version. I'll keep this comment open to remove these changes from the final PR once ready for one final review.